src/app/components/footer/footer.component.scss

```diff
@@ -26,7 +26,7 @@ footer {
   line-height: normal;
   padding: 45px 0 0 0;
   color: #fff;
-  box-shadow: 0px -3px 6px #00000029;
+  box-shadow: 0px -3px 6px rgba(0, 0, 0, 0.1);
```

Comment:
This is the same as the scss variable, no?

Reply:
This is because the direction of the boxshadow is up instead of down.

Reply:
And this is the only place that is neccesary